daktari/checks/android.py

```diff
@@ -0,0 +1,25 @@
+from daktari.check import Check, CheckResult
+from daktari.os import OS, get_env_var_value
+
+
+class AndroidNdkHomeSet(Check):
+    def __init__(self, expected_version):
+        self.name = "env.variableSet.androidNdkHome"
+        self.variable_name = "ANDROID_NDK_HOME"
+        self.expected_version = expected_version
+        self.suggestions = {
```

Comment:
Can we just use EnvVarSet check for this instead of a new check doing what I think is the same thing?

Reply:
It would require extending `EnvVarSet` to check for a partial match (which might be worth doing?), it currently just checks the value is equal to the expected string. 

Also thought it was painful to have to include the wall of suggestion text and flags we'd need to pass into `EnvVarSet` in the list of checks. Having it `AndroidNdkHomeSet("25.2.9519653")` feels nice and readable.